sysdig/resource_sysdig_secure_cloud_auth_account_component_test.go

```diff
@@ -59,6 +59,7 @@ resource "sysdig_secure_cloud_auth_account" "azure_sample" {
   enabled            = true
   provider_tenant_id = "%s"
   provider_alias     = "some-alias"
+  regulatory_framework = "REGULATORY_FRAMEWORK_UNSPECIFIED"
```

Comment:
I don't believe this is necessary. Not specifying the regulatory_framework here will make protobuf automatically set it to REGULATORY_FRAMEWORK_UNSPECIFIED. Basically the meaning of this enum value itself is that the field was not provided/populated in the request.

The ACC tests are failing because of BE changes not available in prod envs. Once they start passing, without this change also they should pass IMO.

Reply:
Yeah, we discussed this so I'll get rid off these